src/richtext/richtextsymboldlg.cpp

```diff
@@ -1034,7 +1034,7 @@ void wxSymbolListCtrl::OnPaint(wxPaintEvent& WXUNUSED(event))
 
 void wxSymbolListCtrl::DoHandleItemClick(int item, int WXUNUSED(flags))
 {
-    if (m_current != item)
+//    if (m_current != item)
```

Comment:
This can't be intentional, can it?

Also, could you _please_ review your PRs before submitting them?

Reply:
@vadz ,
Oops, sorry.

Fixed now.

Thx.